### DISCUSSION TITLE
(launcher) ubuntu20@ubuntu20:~/slurm/slurm-web$ racksdb racks [ERROR] racksdb.exec: Error while loading db: Property site-packages is not defined in schema for object Schema_content

### DISCUSSION THREAD

**Discussion by GoSibo** [rackslab/RacksDB#88] [Q&A] [answered]
Hi ，
     based on Ubuntu20.04 and this guide：
![image](https://github.com/user-attachments/assets/8b5a0049-d543-4839-be4c-bfc7aece10be)
      also ，I typed the conmand: sudo cp -rf /usr/share/doc/python3-racksdb/examples/db/* /var/lib/racksdb/

     when I type :  racksdb racks or other, I encount the problem above, 
     thanks!

**Comment by rezib** [comment, marked as answer]
Ubuntu 20.04 is not supported, can you consider upgrading to Ubuntu 24.04?

**Reply by GoSibo** [reply]
Oh, thanks Rémi, I try add those lines in  /usr/share/racksdb/schemas/racksdb.yml:
 "    site-packages:
      type: list[:Infrastructure]
      description: "Path to the site-packages directory"
"
     this issue fix now , but I am not sure whether some other errors will occur, so I will  upgrading to Ubuntu 24.04, thanks!
